teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(name = 'Unknown', age = nil, parent_permission: true, specialization: nil)
+    super(name, age, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    super || @specialization
+  end
```

Comment:
- [ ] I think this method `can_use_services?` should always return true according to the project requirement.
```suggestion
  def can_use_services?
    true
  end
```